www/assets/js/sofia.data.js

```diff
@@ -341,6 +345,41 @@ S.data = {
                     //TODO handle errors
                 });
                 */
+              },
+              showConfigurationModal: function () {
+                  S.tool.getDialog("#show-config-dialog").showModal();
+              },
+              closeConfigurationModal: function () {
+                  this.db = S.config.db; //Reset
+                  S.tool.getDialog("#show-config-dialog").close();
+              },
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["event" is defined but never used](https://www.codacy.com/app/sapk/SOFIA/file/2059618678/issues/source?bid=2990559&fileBranchId=3135266#l356)